private-templates-service/package.json

```diff
@@ -5,12 +5,24 @@
 	"main": "server.js",
 	"scripts": {
 		"test": "echo \"Error: no test specified\" && exit 1",
-    "start:prod": "npm install && node server.js",
-    "start": "node server.js"
+		"start:prod": "npm install && node server.js",
+		"start": "tsc && node dist/server.js | bunyan"
```

Comment:
Previously it was using spaces but now the whitespace is tabs, can we keep using spaces?

Reply:
Yep sounds good